cmd/gpud/command/update.go

```diff
@@ -43,7 +43,7 @@ func detectLatestVersion() (string, error) {
 		return "", err
 	}
 
-	fetchedVer, err := distsign.Fetch(defaultURLPrefix+track+"_latest.txt", 100)
+	fetchedVer, err := distsign.FetchAndTrimSpace(defaultURLPrefix+track+"_latest.txt", 100)
```

Comment:
it's a bit weird... we should not rely on distsign package here.. we can write this function in the update.go, should call the function `fetchVersion`

Reply:
done